### PR TITLE
Docs: Add/update contributing guidelines specific to this project

### LINKED ISSUE #29: Add/update a CONTRIBUTING.md for contributing guidelines
### Feature details?

A guide on how to contribute to this project would be helpful.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by siddhantk232**
## Description

contributing and code of conduct for this project

### Fixed issue(s): #

fixes #29

### Changes Include

-   [x] documentation

### Checklist

-   [x] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings

### CODE REVIEW

**Comment by siddhantk232** [comment]
The technical guide to set up a dev environment or tools required are trivial for now so I skipped them. We can later update the contribution guide to extend any new rules we add :smile:

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Contributor Code of Conduct
```
Comment (Jarmos-san) [inline CODE_OF_CONDUCT.md:1]: @Bhupesh-V do we need another project-specific `CODE_OF_CONDUCT.md`? The [organization-wide `CODE_OF_CONDUCT.md`](https://github.com/developersIndia/.github/blob/main/CODE_OF_CONDUCT.md) will be used everywhere regardless.

**Review by Jarmos-san — COMMENTED** [review]
Looks good tbh!

Need further clarification on the redundant `CODE_OF_CONDUCT.md` since there's already an organization-wide Code of Conduct available elsewhere.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Bhupesh-V) [reply CODE_OF_CONDUCT.md:1]: No, we don't need another COC, the organisation one is accessible in this repo as well. Let's remove that for now

**Review by Jarmos-san — APPROVED** [review]

**Review by varshney-himanshu — APPROVED** [review]
Looks great!

**Pull request merged**